src/ThreeEditor/js/Loader.js

```diff
@@ -155,7 +155,17 @@ function Loader(editor) {
 				break;
 
 			case 'editor':
-				if (window.confirm('Current editor data will be lost. Are you sure?')) {
+				let versionIsOk = true;
+				if (data.metadata.version !== editor.jsonVersion) {
+					versionIsOk = window.confirm(
+						`File was generated with different version of editor JSON: ${data.metadata.version}\nCurrent version of editor JSON: ${editor.jsonVersion}\nContinue?`
```

Comment:
What about backward and forward compatibility ? Maybe we should use some proper versioning compare methods ?

Reply:
```suggestion
						`File has  project standard version ${data.metadata.version} which is different from standard version handled by this web application: ${editor.jsonVersion}\nContinue?`
```

Reply:
> 

I think we should care about backward compatibility, after the first full release(end of engineer thesis ). And then if JSON will change we could add methods to handle old versions.  

More validation of deserialized JSON and default values when loading data could help with compatibility, but it is another issue.